src/components/BillListItem.vue

```diff
@@ -38,6 +38,14 @@
 				</template>
 				{{ deleteIconTitle }}
 			</ActionButton>
+			<ActionButton
```

Comment:
```suggestion
			<ActionButton v-if="!timerOn"
```
This ActionButton should not be there when we are deleting a bill. This way the cancel button becomes the only one visible so there is no menu so it's easier to see and quicker to click.

Reply:
Fixed